ibraheemosule/skoolizy
# Ios routes fix

# Why this PR?
- My profile page is created
- New reusable components created

# What does this PR do?
- The profile page has been created for teachers
- A reusable list component has been created and it has the edit functionality

# How to test your work?
- Go to my profile from the navigation bar
- See the hard-coded profile details
- Click on the ones with the edit option (this should open the edit modal for that field)

# Pages to see your changes?
- baseUrl/my-profile
